Source/Core/Core/FileMonitor.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright 2009 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/FileMonitor.h"
+
+#include <algorithm>
+#include <cctype>
+#include <memory>
+#include <string>
+#include <unordered_set>
+
+#include "Common/CommonTypes.h"
+#include "Common/Logging/LogManager.h"
```

Comment:
Missed this one the previous review, but `Common/Logging/Log.h` also needs to be included for the logging #defines. After that, it looks fine to merge.